device/mellanox/x86_64-mlnx_msn4800_simx-r0/ACS-MSN4800/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,112 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '26451968' %}
+{% set ingress_lossy_pool_size =  '26451968' %}
+{% set egress_lossless_pool_size =  '60817392' %}
+{% set egress_lossy_pool_size =  '26451968' %}
```

Comment:
How are these numbers calculated? 
If they are copied from other platforms, I suggest using symbol link pointing to the original file.
Otherwise, can you provide the number of uplink and downlink ports?